src/components/UpdateAppModal/BaseUpdateAppModal.js

```diff
@@ -67,4 +67,5 @@ class BaseUpdateAppModal extends PureComponent {
 
 BaseUpdateAppModal.propTypes = _.omit(propTypes, 'version');
 BaseUpdateAppModal.defaultProps = _.omit(defaultProps, 'version');
+BaseUpdateAppModal.propTypes = propTypes;
```

Comment:
Doesn't this just create another lint error where the `version` prop is unused?

I think this particular lint error is incorrect, because `_.omit(propTypes, 'version')` _does_ include `onSubmit`, but it's just not evaluated at compile-time (maybe because it's using underscore).

Reply:
Ah missed this. I agree the `_.omit` seems like the culprit. I am not a `propType` expert, but what is the harm in including it?

Reply:
I think it's actually not a problem because there is `// eslint-disable-next-line react/no-unused-prop-types` in the version prop file. If you didn't have that, then eslint would complain about the unused `version` prop.

Anyways, I suggest just replacing:

```javascript
BaseUpdateAppModal.propTypes = _.omit(propTypes, 'version');
BaseUpdateAppModal.defaultProps = _.omit(defaultProps, 'version');
```

with

```javascript
BaseUpdateAppModal.propTypes = propTypes;
BaseUpdateAppModal.defaultProps = defaultProps;
```

Reply:
Gonna chime in here but I think the convention of using `omit` is a little backwards. If you have some `propTypes` that need to be shared then you should move the common denominator set of `propTypes` to a file and then import it in the two places it's needed. If one of those places also needs the `version` then add it to that one instead of omitting it where it isn't needed.

Reply:
> I think the convention of using omit is a little backwards. If you have some propTypes that need to be shared then you should move the common denominator set of propTypes to a file and then import it in the two places it's needed. If one of those places also needs the version then add it to that one instead of omitting it where it isn't needed.

Makes sense to me 👍